src/main/cpp/commands/drivetrain/DriveSetSpeedCommand.cpp

```diff
@@ -0,0 +1,35 @@
+
+#include "commands/drivetrain/DriveSetSpeedCommand.h"
+
+DriveSetSpeedCommand::DriveSetSpeedCommand(
```

Comment:
should we make this just an overload of DriveCommand, or would that be too hard? I know Java overloads are annoying, no idea about C++

Reply:
We should consider making better commands for next year. We can diagram this out in a design ticket or something I think